drivers/hmis/app/graphql/types/hmis_schema/enums/hud/ad_hoc_yes_no.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+# THIS FILE IS GENERATED, DO NOT EDIT DIRECTLY
+
+module Types
+  class HmisSchema::Enums::Hud::AdHocYesNo < Types::BaseEnum
+    description 'ad_hoc_yes_no'
+    graphql_name 'AdHocYesNo'
+    value 'NO', '(0) No', value: 0
```

Comment:
Will this display as "(0) No", or is there code somewhere to just display "No"?  I feel like we should support both, just want to confirm we're not accidentally including all of the numbers hard coded if we might want to exclude them.

Also, do we like them better at the front or back?  The front makes sense because the length of the code is usually one or two characters, but the end makes sense because then you get the human readable version first.

Reply:
There is codegen code in the frontend to generate Enum descriptions based on this file [here](https://github.com/greenriver/hmis-frontend/blob/fa601d60b060da53d1ce27085b651d54767fbc09/scripts/generateEnumDescriptions.js#L101). [This](https://github.com/greenriver/hmis-frontend/blob/stable/src/types/gqlEnums.ts) is the file we use for all display strings in the interface. If we eventually want to allow toggling them on and off, we could just generate 2 of those files.

I think probably the front, but we can cross that bridge if/when we ever decide to show them to the user?

Reply:
That looks perfect.  Stupid question, but why are we including the code if we just strip it out for display?

Reply:
Ok, I think I get it.  Those end up in the schema, which makes that file way more readable.

Reply:
> why are we including the code if we just strip it out for display?

🤷  I don't know, I thought Zach added the numeric prefixes at your request. But yeah I guess why not have them in the schema, since we may want to show them later